scripts/run-integration.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+#
+# Runs the integration tests
+#
+# Requirements:
+# - a kubernetes cluster with a Windows nodepool
+#
+# Steps:
+# - cross compile the csi-proxy binary and the integration tests
+# - copy to the VM using scp
+# - restart the CSI Proxy binary process with a helper powershell script
+# - run the integration tests
+
+set -euxo pipefail
+
+pkgdir=${GOPATH}/src/github.com/kubernetes-csi/csi-proxy
+source $pkgdir/scripts/utils.sh
+
+main() {
+  compile_csi_proxy
+  compile_csi_proxy_integration_tests
+  sync_csi_proxy
+  sync_csi_proxy_integration_tests
+  sync_powershell_utils
+  restart_csi_proxy
```

Comment:
why restart? it was already started?

Reply:
the idea is to test the latest binary and not the one that's preinstalled and running in the node, `sync_csi_proxy` only copies the binary to the host, `restart_csi_proxy` restarts it